upgrade/upgrade-scripts/upgrade-container

```diff
@@ -648,6 +649,28 @@ function migrate_configuration() {
 		/var/target/pr
 	EOF
 
+	#
+	# These directories and files are created by cloud-init. When
```

Comment:
Can you also explain, somewhere in this comment, why we don't want cloud-init to re-generate this information after a not-in-place upgrade.

After reading this comment, the PR description, and the bug report, I still don't understand why we can't rely on cloud-init to properly regenerate this information after the upgrade.

Reply:
I've added more comment.